src/main/resources/application.yaml

```diff
@@ -55,3 +55,5 @@ scheduling:
     delete-dispatched-files:
       enabled: true
       cron: ${DELETE_DISPATCHED_FILES_CRON}
+    scan:
+      delay: 30000 # In milliseconds
```

Comment:
do we need 30 secs delay by default. Better to configure this per environment where AAT/Preview can have minimum delay ?

Reply:
there was no need to customise this property. at the time of creating this pr functional test configuration was still underway and this property can be simply included into `values.aat` and `values.preview` without amending template which would need chart version upgrade

Reply:
we can at least add env var here so that (clear) overwriting is possible

Reply:
It needs an env var else you will have to bootstrap from vault which is not required for this property.